prov/tcp/src/tcpx.h

```diff
@@ -155,6 +155,7 @@ struct tcpx_ep {
 	fastlock_t		lock;
 	tcpx_ep_progress_func_t progress_func;
 	bool			send_ready_monitor;
+	struct sockaddr_in	*dest_addr;
```

Comment:
We need to support ipv6.  But do we really need to store this with the EP, versus just retrieving it from the socket when needed?